pkg/astrolabe/protected_entity_info.go

```diff
@@ -37,17 +38,19 @@ type ProtectedEntityInfo interface {
 type ProtectedEntityInfoImpl struct {
 	id                 ProtectedEntityID
 	name               string
+	size               int64
```

Comment:
use sizeInBytes instead or add a comment to call it out?

Reply:
Added a comment, thanks